notebooks/02_model/mmlspark_lightgbm_criteo.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>Shouldn't you read the version of mmlspark from scripts/databricks_install.py ?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>